### DISCUSSION TITLE
In Line command Execution

### DISCUSSION THREAD

**Discussion by friosavila** [hugetim/nbstata#30] [Ideas]
Hi Tim
I was wondering if you have considered modifying nbstata for "in line command"
Contrary to a previous comment I had before, it seems that line command execution is rather new. 
see [here](https://quarto.org/docs/get-started/computations/rstudio#inline-code).

I tried doing the same with nbStata 
Something like `stata display "hola"`. And seems that it tries to call on Stata, but something gets missing.

of course, I have no idea how that could be done, but could be nice to have.

Thank you!
Fernando

**Comment by hugetim** [comment]
Do you have any more information about how Quarto does that under the hood, such as what languages it is supported for? If it works via a Jupyter kernel, we should be able to support it. But if it's via [Knitr](https://quarto.org/docs/computations/execution-options.html#engine-binding), then probably not.
